src/main/java/com/exasol/adapter/dialects/elasticsearch/ElasticSearchMetadataReader.java

```diff
@@ -0,0 +1,32 @@
+package com.exasol.adapter.dialects.elasticsearch;
+
+import java.sql.Connection;
+
+import com.exasol.adapter.AdapterProperties;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.jdbc.*;
+
+public class ElasticSearchMetadataReader extends AbstractRemoteMetadataReader {
```

Comment:
Add javadoc to all public and protected  classes, methds and interfaces. Only exception: visitors there you don't need javadoc for the visit methods

Reply:
and for @override methods also not

Reply:
and not for tests ;-)